src/stores/Binding/Store.ts

```diff
@@ -926,6 +931,16 @@ const getInitialState = (
         );
     },
 
+    setOnIncompatibleSchemaChangeErrorExists: (value, key) => {
+        set(
+            produce((state: BindingState) => {
+                state.onIncompatibleSchemaChangeErrorExists[key] = value;
+            }),
+            false,
+            'On Incompatible Schema Change Error Exists Set'
+        );
+    },
```

Comment:
100% do not want to handle this in the PR. However, I think we should not have `scope` be involved with setting this and we should eventually add this to the meta data of the binding.

That way the new fields can also show an error icon in the selector.
![image](https://github.com/user-attachments/assets/af8abf77-7302-4642-9287-aea10b793835)